BranchSDK/src/BranchIO/AdvertiserInfo.h

```diff
@@ -82,6 +82,9 @@ class BRANCHIO_DLL_EXPORT AdvertiserInfo : public PropertyManager {
     bool trackingDisabled;
     bool trackingLimited;
 
+    static constexpr const char *const AdvertiserStorage = "advertiser";
```

Comment:
variable that starts with a capital letter

Reply:
Fixed